Assets/Scripts/interactableProvider.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class interactableProvider : Interactable
+{
+
+    private int ACTION_PROVIDE = 1;
+    private int ACTION_NOTHING = 0;
+    // Start is called before the first frame update
+
+    private infoProvider provider;
+
+    void Start()
+    {
+        provider = this.GetComponent<infoProvider>();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    override public string getInstructions(int player_id)
+    {
+        if (player_id == provider.getRestrict())
+        {
+            return "Press B to get information";
```

Comment:
think there's a way to have it display the button dynamically from the rewired controller? It might get confusing if we don't only use XBOX controllers.